src/Simplex/Chat/Controller.hs

```diff
@@ -301,7 +301,7 @@ data ChatCommand
   | APIGetAppSettings (Maybe AppSettings)
   | APIGetChatTags UserId
   | APIGetChats {userId :: UserId, pendingConnections :: Bool, pagination :: PaginationByTime, query :: ChatListQuery}
-  | APIGetChat ChatRef (Maybe MsgContentTag) ChatPagination (Maybe String)
+  | APIGetChat ChatRef (Maybe GroupChatFilter) ChatPagination (Maybe String)
```

Comment:
I don't understand why it is needed. If you don't want to support MsgContentTag filter in support chats, then you simply pass nothing here, but chatscope is already in chat ref, so why it's here too?

Reply:
yes, it's redundant, there is a comment - didn't get to it (I added scope to chat ref after this was already working)